docs/troubleshooting.md

```diff
@@ -1,8 +1,43 @@
-Troubleshooting [WIP]
+# Troubleshooting
 
-* How to view Ingress Controller logs ?
 
+The Application Gateway Ingress Controller relies primarily on the Kubernetes
+[Service](https://kubernetes.io/docs/concepts/services-networking/service/) and
+[Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) resources to construct
+configuration for App Gateway. Surprising AGIC behavior (or none at all) could be as a result of
+missing or incorrect configuration.
 
-* How to check Pods health
-* How do you get the current config
-* How do enable verbose logging
\ No newline at end of file
+
+* Get the namespaces you currently have on your Kubernetes cluster:
+```bash
+kubectl get namespaces
+```
+
+
+* Is the AGIC pod in the `default` namespace?  Is it healthy? In `STATUS` column you should see
+`Running`, the `NAMESPACE` column should be `default`. You should have exactly one AGIC pod.
```

Comment:
It might be wrong to assume that customer will have one AGIC pod.

Reply:
Discussed offline. This is now `at least one`